scripts/files/fs_local.py

```diff
@@ -20,3 +25,30 @@ def read(path: str) -> bytes:
     """
     with open(path, "rb") as file:
         return file.read()
+
+
+def rename(path: str, new_path: str) -> None:
+    """Rename a file.
+
+    Args:
+        path (str): original path
+        new_path (str): path to be renamed with
+    """
+    if not os.path.exists(path):
```

Comment:
checking if a file exists before doing something with it is somewhat of a antipattern.

between the time you check and the time you access it, the file could have been moved.

you are best to handle it in the actual action `os.rename` (Im assuming it throws if the file doesnt exist?

does it also throw if the target does exist?

Reply:
Looks like it does manage those: https://docs.python.org/3/library/os.html#os.rename

Reply:
> does it also throw if the target does exist?
I was looking at this now and

It does override the file as per
> If both are files, dst will be replaced silently if the user has permission.

I'm not too sure of what behaviour we want to have for the general functionality? 
I think for our context of renaming the tiff files as we don't want any duplicate we should probably do a check of duplicate files before the process of validate the tiff anyway?